### DISCUSSION TITLE
What would be a good get started example in SAM CLI?

### DISCUSSION THREAD

**Discussion by heitorlessa** [aws-powertools/powertools-lambda-python#359] [Q&A] [answered]
Hello folks!

We're looking to include an example within SAM CLI for customers using `sam init - We need your help to decide what to create.

Ideally, it would be something simple to get started and understand, but not as trivial as a Hello World return. We can also be opinionated when it comes to tooling to help those who haven't created a Python project before - e.g., black, isort, pre-commit, etc.

Thank you!

**Comment by mwarkentin** [comment, marked as answer]
Some "classic" lambda uses could be:

- SNS -> slack webhook
- s3 image manipulation (thumbnail, crop, watermark, etc)
- some sort of cron function (turn off my RDS instances at 5pm)

**Comment by MrTomerLevi** [comment]
Agree with @mwarkentin and wants to add a few more options:

- API Gateway + Lambda - where lambda include an `ApiGatewayResolver`, logging and tracing 
- SQS + Lambda - where lambda include `@sqs_batch_processor`, logging and tracing

**Comment by heitorlessa** [comment]
Thanks a lot for the additional ideas, Tomer — we’ve got the first one
already out

sam init —location
https://github.com/aws-samples/cookiecutter-aws-sam-python
